### PR TITLE
OCPBUGS-17409 Adding link to KB article 

### PR DESCRIPTION
**Pull request by kquinn1204**
[OCPBUGS-17409]: 

Version(s): 4.10,4.11,4.12,4.13,4.14, and main

Issue: https://issues.redhat.com/browse/OCPBUGS-17409

Link to docs preview: https://64297--docspreview.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-sriov-operator.html#about-sr-iov-operator-admission-control-webhook_configuring-sriov-operator

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kquinn1204** [comment]
@SchSeba Sorry just got round to this. I thought I had created the PR are you happy with this change, it has a callout to the KB ?

**Review by SchSeba — COMMENTED** [review]
/lgtm

File: modules/nw-sriov-configuring-operator.adoc [hunk 1]
```diff
@@ -110,7 +110,7 @@ Admission Controller application. It provides the following capabilities:
 
 By default the SR-IOV Network Operator Admission Controller webhook is enabled by the Operator and runs as a daemon set on all control plane nodes.
 
-NOTE: Use caution when disabling the SR-IOV Network Operator Admission Controller webhook. You can disable the webhook under specific circumstances, such as troubleshooting, or if you want to use unsupported devices.
+NOTE: Use caution when disabling the SR-IOV Network Operator Admission Controller webhook. You can disable the webhook under specific circumstances, such as troubleshooting, or if you want to use unsupported devices. For information on configuring unsupported devices, see link:https://access.redhat.com/articles/7010183[Configuring the SR-IOV Network Operator to use an unsupported NIC].
```
Comment (tmalove) [inline modules/nw-sriov-configuring-operator.adoc:113]:
```suggestion
NOTE: Use caution when disabling the SR-IOV Network Operator Admission Controller webhook. You can disable the webhook under specific circumstances, such as troubleshooting, or if you want to use unsupported devices. For information about configuring unsupported devices, see link:https://access.redhat.com/articles/7010183[Configuring the SR-IOV Network Operator to use an unsupported NIC].
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

**Review by tmalove — APPROVED** [review]

File: modules/nw-sriov-configuring-operator.adoc [hunk 2]
```diff
@@ -110,7 +110,7 @@ Admission Controller application. It provides the following capabilities:
 
 By default the SR-IOV Network Operator Admission Controller webhook is enabled by the Operator and runs as a daemon set on all control plane nodes.
 
-NOTE: Use caution when disabling the SR-IOV Network Operator Admission Controller webhook. You can disable the webhook under specific circumstances, such as troubleshooting, or if you want to use unsupported devices.
+NOTE: Use caution when disabling the SR-IOV Network Operator Admission Controller webhook. You can disable the webhook under specific circumstances, such as troubleshooting, or if you want to use unsupported devices. For information about configuring unsupported devices, see link:https://access.redhat.com/articles/7010183[Configuring the SR-IOV Network Operator to use an unsupported NIC].
```
Comment (mjpytlak) [inline modules/nw-sriov-configuring-operator.adoc:113]:
```suggestion
[NOTE]
====
Use caution when disabling the SR-IOV Network Operator Admission Controller webhook. You can disable the webhook under specific circumstances, such as troubleshooting, or if you want to use unsupported devices. For information about configuring unsupported devices, see link:https://access.redhat.com/articles/7010183[Configuring the SR-IOV Network Operator to use an unsupported NIC].
====
```
@kquinn1204 While your admonition publishes correctly, our guidance has using this syntax.

Comment (kquinn1204) [reply modules/nw-sriov-configuring-operator.adoc:113]: My bad I was just updating previous note and I should have spotted that. Thanks

**Comment by kquinn1204** [comment]
@mjpytlak Should be good for merging now

**Comment by mjpytlak** [comment]
Good to go @kquinn1204 have a good weekend.

**Pull request merged**